com.ugetdm.uGet.yaml

```diff
@@ -0,0 +1,119 @@
+app-id: com.ugetdm.uGet
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: uget-gtk
+rename-icon: uget-icon
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  # Audio access
+  # for sound notifications
+  - --socket=pulseaudio
+  # Filesystem access
+  - --filesystem=xdg-download
+  # Network access
+  - --share=network
+  # Notifications
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # OpenGL access
+  - --device=dri
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: aria2
+    config-opts:
+      - --enable-bittorrent
+      - --enable-metalink
+      - --enable-websocket
+      - --enable-epoll
+      # GnuTLS has precedence over OpenSSL if both libraries are installed.
+      - --without-openssl
+      - --with-gnutls
+      - --with-libcares
+      - --with-libxml2
+      - --with-libz
+      - --with-sqlite3
+      - --enable-gnutls-system-crypto-policy
+      - --with-ca-bundle=/etc/ssl/certs/ca-certificates.crt
+    sources:
+      - type: archive
+        url: https://github.com/aria2/aria2/releases/download/release-1.37.0/aria2-1.37.0.tar.xz
+        sha256: 60a420ad7085eb616cb6e2bdf0a7206d68ff3d37fb5a956dc44242eb2f79b66b
+        x-checker-data:
+          type: anitya
+          project-id: 109
+          url-template: https://github.com/aria2/aria2/releases/download/release-$version/aria2-$version.tar.xz  
+    post-install:
+      - install -p -D -m 0644 "COPYING" LICENSE* -t "${FLATPAK_DEST}/share/licenses/aria2/";
+
+  - name: c-ares
```

Comment:
c-ares and libssh2 must be before aria2

Reply:
that's the buile error, but as I mentionned I don't see where these dependencies are needed.

Reply:
afaik aria2c is one of the alternative backends ugetdm supports